### PR TITLE
redfish modules: documentation language and formatting

### PR DESCRIPTION
**Pull request by bluikko**
##### SUMMARY

Update the documentation block with more English language
more consistent with other modules; finish each description
with a full stop and start with capital case letter.

Use consistent language and terminology within each module/group
of modules.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

all remote_management/redfish plugins

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/remote_management/redfish/redfish_command.py [hunk 1]
```diff
@@ -63,28 +63,28 @@
...
     aliases: [ account_roleid ]
     description:
-      - Role of account to add/modify
+      - Role of account to add/modify.
     type: str
   bootdevice:
     required: false
     description:
-      - bootdevice when setting boot configuration
+      - Bootdevice when setting boot configuration.
```
Comment (felixfontein) [inline plugins/modules/remote_management/redfish/redfish_command.py:83]: Is bootdevice (without space) a domain-specific term, or should it be two words?
```suggestion
      - Boot device when setting boot configuration.
```

**Review by felixfontein — COMMENTED** [review]
Thanks a lot for improving the docs!

**Comment by mraineri** [comment]
All looks good to me!

shipit

**Comment by felixfontein** [comment]
@bluikko thanks for improving the docs!
@mraineri thanks for reviewing this!

**Pull request merged**